### DISCUSSION TITLE
How do i install discord.py for my discord bot?

### DISCUSSION THREAD

**Discussion by HorrorSpirit** [atom/atom#22533] [Q&A] [answered]
I already have the code for my bot, but when i try to run it it gives me a syntax error telling me that the discord.py syntax is wrong, even though it is right and atom simply doesn't recognize it. How do i install discord.py on atom to be able to host my bot from atom?

**Comment by ThatXliner** [comment, marked as answer]
Read [this](https://discordpy.readthedocs.io/en/stable/intro.html). This is not an issue with Atom specifically

**Reply by HorrorSpirit** [reply]
It says i already have it, yet the code still won't work. Is there an extra step i am missing like adding that to atom or something?

**Reply by ThatXliner** [reply]
How are you running it? Are you on the command line? What OS?

**Reply by HorrorSpirit** [reply]
I'm on Mac, on the terminal app thing that is there. I have just updated to Big Sur if that affects anything.

**Reply by ThatXliner** [reply]
What is the script you are running to run the bot? 

> Is there an extra step i am missing like adding that to atom or something?

No, this is not specifically Atom-related, AFAIK

**Reply by HorrorSpirit** [reply]
Seems like this isn't the best place to look for an answer then. Thanks for taking the time anyways!
